### ISSUE TITLE
FreeCodeCamp Issue Template - a suggestion

### ISSUE THREAD

**Issue by Tzahile** [FreeCodeCamp/FreeCodeCamp#9357]
Under the title **Your Code**, there's a place for pasting js code:
```html
#### Your Code

```js
// If relevant, paste all of your challenge code in here

.```
```

Note that the comment:
_// If relevant, paste all of your challenge code in here_
should be **before** the js-code apostrophes.

**Comment by adamisntdead** [comment]
It is clearer, for those not familiar with markdown, where to put the code if its like that, So I would suggest not changing it?

**Comment by Tzahile** [comment]
This is only a suggestion of course. people aren't stupid and it's possible to write something like:
``
// If relevant, paste all of your challenge code under the next line
``
Then the comment won't be shown in public

**Comment by raisedadead** [comment]
@Tzahile @adamisntdead thanks, I don't think this will hurt. Let's get this done.

File to be edited
https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/.github/ISSUE_TEMPLATE.md

and should have the line
```
 // If relevant, paste all of your challenge code after the next line
```

**Comment by kalila855** [comment]
I can take it!

**Comment by raisedadead** [comment]
Resolved in FreeCodeCamp/FreeCodeCamp#9374
